ComposeSamples/ListDetail/src/main/java/com/microsoft/device/display/samples/listdetail/models/AppStateViewModel.kt

```diff
@@ -14,11 +14,11 @@ class AppStateViewModel : ViewModel() {
     private val imageSelectionLiveData = MutableLiveData<Int>() // observe the image selection change
     private val isDualModeLiveData = MutableLiveData<Boolean>() // observe the dual-screen mode
 
-    fun getIsDualModeLiveDataLiveData(): LiveData<Boolean> {
```

Comment:
instead using getters and setters you could follow a more kotlin approach where isDualModeLiveData is public and there is need of such these properties.